gateway/engine/policies/pom.xml

```diff
@@ -113,6 +113,13 @@
           </instructions>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>-Xmx5g</argLine>
```

Comment:
Do we really need such a big max value for memory?

Reply:
Yes, it was OOMing on CI because the Apache LDAP server used during testing spikes the heap size briefly. I reached this number after trying a few different values. This is just a ceiling number (mx = heap memory maximum), and the JVM will only use that much if it needs to.